rhaegal_data_params.yaml

```diff
@@ -0,0 +1,6 @@
+remote_user: "your_remote_user"
+remote_temp_directory: "/mnt/mydrive/rosbags/temp"

```

Comment:
We do not need to explicetly parametise this directory as it can be creted from `remote_directory`. This way we can always do `mkdir remote_directory/temp`

Reply:
deleted remote_temp_directory and modified the logic

Reply:
:eyes: